affiliates/argentina.swagger.json

```diff
@@ -268,6 +268,12 @@
           }
         },
         "Agreements" : {
+          "type" : "object",
+          "additionalProperties" : {
+            "$ref" : "#/components/schemas/Agreement"
```

Comment:
![image](https://user-images.githubusercontent.com/3830714/204248077-66b38052-db45-4d76-a14c-1aa902817b2a.png)

I'm afraid additionalProperties don't work the way you think they work

Reply:
If all the properties should be defined in admin then now it should be fine.

Reply:
examples for properties

Reply:
![image](https://user-images.githubusercontent.com/4953629/204324094-2c423dc7-628d-46ce-bf00-41c2b53bfec8.png)

Reply:
fixed

Reply:
no more required arrays